lib/src/base-styles/_nx-card.scss

```diff
@@ -6,176 +6,74 @@
  */
 
 .nx-card-container {
+  column-gap: $nx-spacing-md;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
   margin: $nx-spacing-md 0;
+  row-gap: $nx-spacing-md;
+}
+
+.nx-card-container--no-wrap {
+  flex-wrap: nowrap;
 }
 
 .nx-card {
+  @include container-vertical;
   background-color: var(--nx-component-background-color);
   border-radius: $nx-border-radius;
   box-sizing: border-box;
-  margin-bottom: $nx-spacing-md;
-  padding: $nx-spacing-l $nx-spacing-xxl;
+  display: flex;
+  flex-direction: column;
+  padding: $nx-spacing-l;
+  text-align: center;
+  min-width: 230px;
+  max-width: 330px;
 }
 
-.nx-card__call-out {
-  @include bold();
-
-  font-size: 32px;
-
-  .nx-icon--card-call-out {
-    font-size: 48px;
-  }
-
-  .nx-icon--card-call-out-xl {
-    font-size: 64px;
-  }
+.nx-card__header {
+  @include container-vertical;
+  margin-bottom: $nx-spacing-md;
 }
 
-.nx-card-container--row {
+.nx-card__content {
+  align-items: center;
   display: flex;
-  -ms-flex-wrap: wrap;
-  flex-wrap: wrap;
-  margin-bottom: 0;
-
-  + .nx-card-container--row {
-    margin-top: 0;
-  }
-
-  .nx-card {
-    @include container-vertical;
-    display: flex;
-    flex-direction: column;
-    -ms-flex-pack: justify;
-    justify-content: space-between;
-    margin-right: $nx-spacing-md;
-    text-align: center;
-    min-width: 230px;
-    max-width: 330px;
-  }
-
-  .nx-card__header {
-    @include container-vertical;
-    margin-bottom: $nx-spacing-md;
-  }
-
-  .nx-card__call-out {
-    + .nx-card__content {
-      margin-top: $nx-spacing-md;
-    }
-  }
-
-  .nx-card__call-out--text-only {
-    + .nx-card__content {
-      margin-top: $nx-spacing-xs;
-    }
-  }
-
-  .nx-card__footer {
-    border-top: $nx-border-default;
-    margin-top: $nx-spacing-l;
-    padding-top: $nx-spacing-l;
-    width: 100%;
-
-    > .nx-btn-bar {
-      justify-content: center;
-    }
-  }
+  flex-direction: column;
+  flex-grow: 1;
+  gap: $nx-spacing-md;
 }
 
-.nx-card-container--column {
-  @include container-vertical;
+.nx-card__footer {
+  border-top: $nx-border-default;
+  margin-top: $nx-spacing-l;
```

Comment:
IMO this margin would be better placed on the bottom of `nx-card__content`.  As it is here, if you for some reason had a card with only a header and a footer, the margins of the two double up in the middle.

Reply:
> IMO this margin would be better placed on the bottom of `nx-card__content`. As it is here, if you for some reason had a card with only a header and a footer, the margins of the two double up in the middle.

Unlikely but worth doing: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/14b0b0f44eacfc4d9dfa3f9b703f2a046ee0ef63